### ISSUE TITLE
Lodash as dependency

### ISSUE THREAD

**Issue by theaqua** [michaelcontento/redux-storage#2]
```
Error: Cannot find module 'lodash' from '/Users/e/dev/yard/frontend/node_modules/redux-storage/lib'
```

And do you really need entire lodash? You can include each functions separately, `import { merge } from 'lodash'`.

**Comment by michaelcontento** [comment]
Thank you for reporting this! 

You're right that single function imports should be favoured over full module imports (fixed in 74ed8a304c78ba06f8718a4338aa592301d3f942). But the actual bug was caused by adding `lodash` as the wrong kind of dependency (fixed in 35d9eae33b5b76c879ba98398ce9bd754144e6bc). 

Version 1.1.3 is published to npmjs.org :)

**Comment by theaqua** [comment]
Cool, thanks for fast fixes!
